tests/testthat/_snaps/Design-methods.new.md

```diff
@@ -1144,98 +1144,75 @@
     BAAJAAAAB3NpZ21hMlcABAAJAAAAA3JobwAEAAkAAAAJdXNlX2ZpeGVkAAQACQAAAAlkYXRh
     bmFtZXMABAAJAAAAD2RhdGFuYW1lc19wcmlvcgAEAAkAAAAGc2FtcGxlAAAEAgAAAf8AAAAQ
     AAAAAgAEAAkAAAASdGJsX0R1YWxFbmRwb2ludFJXAAQACQAAAARsaXN0AAAA/gAAAxMAAAAK
-    AAADEwAAAAEAAAAOAAAAAAAABAIAAAH/AAAAEAAAAAMABAAJAAAABnRibF9kZgAEAAkAAAAD
-    dGJsAAQACQAAAApkYXRhLmZyYW1lAAAEAgAAAv8AAAANAAAAAAAABAIAAAP/AAAAEAAAAAEA
-    BAAJAAAAAXcAAAD+AAADEwAAAAEAAAAOAAAAAAAABAIAAAH/AAAAEAAAAAMABAAJAAAABnRi
-    bF9kZgAEAAkAAAADdGJsAAQACQAAAApkYXRhLmZyYW1lAAAEAgAAAv8AAAANAAAAAAAABAIA
-    AAP/AAAAEAAAAAEABAAJAAAAAXgAAAD+AAADEwAAAAEAAAANAAAAAAAABAIAAAH/AAAAEAAA
-    AAMABAAJAAAABnRibF9kZgAEAAkAAAADdGJsAAQACQAAAApkYXRhLmZyYW1lAAAEAgAAAv8A
-    AAANAAAAAAAABAIAAAP/AAAAEAAAAAEABAAJAAAAAXkAAAD+AAADEwAAAAEAAAAOAAAACz/w
-    AAAAAAAAQAgAAAAAAABAFAAAAAAAAEAkAAAAAAAAQC4AAAAAAABANAAAAAAAAEA5AAAAAAAA
-    QEQAAAAAAABASQAAAAAAAEBUAAAAAAAAQFkAAAAAAAAAAAQCAAAB/wAAABAAAAADAAQACQAA
+    AAAADQAAAAAAAAANAAAAAAAAAA4AAAAAAAAADQAAAAAAAAAKAAAAAAAAAAoAAAAAAAAADQAA
```

Comment:
I think we should not have a `*.new.*` file in the PR, you would need to review/accept the snapshot change via `testthat`

Reply:
@Puzzled-Face this file is still here